### DISCUSSION TITLE
Filter Index: localized manyToMany relation getter

### DISCUSSION THREAD

**Discussion by hethehe** [coreshop/CoreShop#2672] [Q&A] [answered]
I'm struggling to find the right index (filter) configuration for my need.
Is it possible to define an index that indexes the localized getter from a relation.

e.g: We have two entities Product and Category
I would like to index the product with the all the names of the categories (which are localized).

I think it could be possible with a nestedLocalized interpreter but I didn't found the right configuration yet.

Is it possible or do I need to create a custom interpreter for this?

Thanks in advance

**Comment by dpfaffenbauer** [comment]
> Is it possible or do I need to create a custom interpreter for this?

@hethehe  Yes.

**Reply by hethehe** [reply]
@dpfaffenbauer  Thanks for the quick reply

Just to be sure. 
Yes I have to create a custom interpreter, or 
yes it's possible with the default interpreters?

**Reply by dpfaffenbauer** [reply, marked as answer]
sorry for the confusion. You have to create a custom Localized Interpreter.
